tutorials/shaders/compute_shaders.rst

```diff
@@ -0,0 +1,242 @@
+.. _doc_compute_shaders:
+
+Using ComputeShaders
+====================
+
+Intro
+-----
+
+Unlike most shaders compute shaders don't have a defined functionality and thus are independent from the graphics pipeline.
+We can think of them as blocks of code executed on the GPU for about any purpose we want.
+The big benefit compared to code executed on a CPU is the high amount of parallelization that GPUs provide.
+
+But because compute shaders are independent of the graphics pipeline we don't have any user defined inputs or outputs. Instead they make changes directly on the GPUs memory which we can read and write  using scripts.
```

Comment:
```suggestion
Because compute shaders are independent of the graphics pipeline we don't have any user defined inputs or outputs (like a mesh going into the vertex shader or a texture coming out of a fragment shader). Instead compute shaders make changes directly on the GPUs memory which we can read and write  using scripts.
```

Reply:
I think there's an accidental double space in `we can read and write using scripts`.